e2-studio-star-rx72n-firmware/src/rx_clock_power_init.c

```
@@ -543,53 +533,44 @@ static rx_err_t internal_module_stop_init(void)
-  uint32_t                         sckcr;
-  uint16_t                         sckcr3;
-  uint8_t                          ppllcr2;
-  uint8_t                          oscovfsr;
-  uint8_t                          memwait;
 
   /* Verify system register access */
   RX_CHECK_NULL_PTR(sys, s_tag, "system_regs pointer is nullptr");
 
   /* Verify PLL is enabled by checking PLLCR2 register */
-  pllcr2 = sys->pllcr2;
+  uint8_t pllcr2 = sys->pllcr2;
```

Clang (14.0.6)

[warning] variable 'pllcr2' is not initialized

(cppcoreguidelines-init-variables)